### DISCUSSION TITLE
More flexible project Iteration type

### DISCUSSION THREAD

**Discussion by khacminh** [community/community#37210] [Projects and Issues]
Hello,

In my opinion, the project's **iteration** field type is not flexible and friendly for project management:

- The iteration is lack of description
- The start/end date of the iterations must not be overlapped. I understand that github team wants to automatically set an iteration to completed. However, in project management, we usually have to schedule the releases overlap to others. For examples:
  - When the development team starts the development, the PO/BA team can start working on the features of the next release
  - When the QA team starts the validation, the development team can start developing the next release
 
So, I think it would be great if github team can add a new field type  for the project with the following features:

- Ability to add markdown description
- The start/end date of the them can be overlapped
- Manually complete/re-open them

Thank you

**Comment by queejie** [comment]
Overlapping dates is common and necessary.

**Reply by DNin01** [reply]
Yeah, this is one of Milestones' current advantages. I wish iterations covered a couple more potential scheduling strategies; as they are now, iterations are designed for more linear schedules.

**Comment by sericaer** [comment]
Markdown description of iteration is important.
